src/app/workspace/workspace-chooser/workspace-chooser.component.ts

```diff
@@ -62,6 +65,10 @@ export class WorkspaceChooserComponent implements OnInit {
   public activeOrgIx: number;
   public activeProjIx: number;
 
+  private organizationsState: OrganizationsState;
+
+  private projectsState: ProjectsState;
```

Comment:
Don't do this! These states are not intended to be used directly in the components. You need to use the right selector for the data you want to get from there.